deployment-aws/terraform/loadbalancer.tf

```diff
@@ -6,7 +6,6 @@ resource "aws_lb" "api" {
   security_groups = [aws_security_group.load_balancer.id]
   access_logs {
     bucket  = "webmev-logs"
```

Comment:
Since we have the new stack-specific bucket, let's change this to `aws_s3_bucket.logging.id`?

Reply:
Yes, good catch.